crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -825,14 +825,24 @@ fn handle_bank_redirects_sync_response(
             reason: reason_info.reason.reject_reason,
             status_code,
             attempt_status: None,
-            connector_transaction_id: None,
+            connector_transaction_id: Some(
+                response
+                    .payment_information
+                    .references
+                    .payment_request_id
```

Comment:
for bank redirects do we get payment_request_id instead of instance_id?

Reply:
Yes we use bank payment_request_id instead of instance_id for bank redirects. Ref `page.no 333` of `trustpay.rs`